test/recipes/03-test_internal_sm3.t

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env perl
+# Copyright 2017-2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This should also be 2021 only since this file is added in 2021

Reply:
It has been modified!